src/state/slices/backdrop_slice.ts

```diff
@@ -27,7 +33,12 @@ type BackdropSliceState = {
   objectOpacity: number;
 };
 
-type BackdropSliceActions = {
+export type BackdropSliceSerializableState = Pick<
+  BackdropSliceState,
+  "backdropVisible" | "backdropBrightness" | "backdropSaturation" | "objectOpacity"
+>;
```

Comment:
Every slice now has a type representation for the subset of state that's serializable! This is used in a few places, including in the serialization/deserialization logic below.

In this case, currently every value of `BackdropSliceState` is serializable. That's not the case for every slice, since some have internal state values or state that is expected to be transient, like the export settings in `ConfigSlice`. 

I use `Pick` here deliberately (rather than setting `BackdropSliceSerializableState = BackdropSliceState`) so that I don't forget to set up serialization for new state values.